# Allows buses to give information to bus central system

bool end_of_route
int32 bus_id
int32 current_node
int32[] responsible_nodes